### ISSUE TITLE
Switch to cli-table

### ISSUE THREAD

**Issue by ColemanGariety** [ColemanGariety/galileo#3]
cli-table looks prettier than terminal table, and terminal table is awfully buggy.

This is cli-table:

![687474703a2f2f692e696d6775722e636f6d2f73597134542e706e67](https://f.cloud.github.com/assets/1356828/1997937/91a81452-852c-11e3-8024-5aba94a0df4e.png)

http://github.com/LearnBoost/cli-table

**Comment by cp** [comment]
Are you going to switch to Node?

**Comment by ColemanGariety** [comment]
@ColbyAley nope, but I can generate the table with node and send it to Ruby. I'd rather find this same style table in Ruby but I have yet to uncover it.

**Comment by ColemanGariety** [comment]
@ColbyAley do you think this can be done via the ruby racer? https://github.com/cowboyd/therubyracer

**Comment by ColemanGariety** [comment]
And then there's this:

https://github.com/wbailey/command_line_reporter

**Comment by cp** [comment]
Command Line Reporter looks nice, even though the name is _not_ nice. I've never used ruby racer outside of Rails, but using it here sounds pretty hacky IMO.

I am dumb. :stuck_out_tongue:

**Comment by ColemanGariety** [comment]
Command Line Reporter apparently doesn't automatically calculate text wrap-around like cli-table and terminal-table do. So this is on hold.

![screen shot 2014-01-31 at 12 44 23 am](https://f.cloud.github.com/assets/1356828/2048561/a5c4a896-8a4b-11e3-8d7e-ac0c9ef7ef3c.png)

If anyone wants to work on this, the code is in the branch 2fa-and-cli-table
